app/src/message/message-reactions.component.js

```diff
@@ -47,7 +49,7 @@ export default function MessageReactions(props) {
           style={{
             cursor: currentUserReactionId ? "pointer" : "default",
           }}
-          title={users.slice(0, MAX_VISIBLE_USERS).join("\n") + (users.length > MAX_VISIBLE_USERS ? `\nand ${users.length - MAX_VISIBLE_USERS} more...` : "")}
+          title={users.slice(0, MAX_VISIBLE_USERS).join("\n") + (users.length > MAX_VISIBLE_USERS ? `\nand ${users.length - MAX_VISIBLE_USERS} ${t("more")}...` : "")}
```

Comment:
The `and` is missing from the translation. I think that you should use [interpolation](https://www.i18next.com/translation-function/interpolation#basic) here

Reply:
Ah sorry, dumb mistake. I completely missed that the "and" was there. Should be fixed now.